<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiMainWindow</class>
 <widget class="QMainWindow" name="uiMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <widget class="QWidget" name="uiCentralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QPushButton" name="exportButton">
      <property name="text">
       <string>Export xml</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="importButton">
      <property name="text">
       <string>Import xml</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="4">
     <widget class="XmlTreeView" name="uiTree">
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="autoScrollMargin">
       <number>3</number>
      </property>
      <property name="rootIsDecorated">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Tree View</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Text View</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" rowspan="4">
     <widget class="QPlainTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XmlTreeView</class>
   <extends>QTreeView</extends>
   <header>pyqtetreemodel</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
